datasets/sk/public_officials/crawler.py

```diff
@@ -0,0 +1,99 @@
+import re
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def parse_details(context: Context, link_el):
+    href = link_el.get("href")
+    name_raw = link_el.text_content().strip()
+    # Filter out non-personal links
+    if re.search(r"#section_[A-Za-z]+$", href) or href.endswith("ViewType=2#top"):
+        return
+    doc = context.fetch_html(href, cache_days=2)
+    table = doc.find(".//table[@class='oznamenie_table']")
+    if table is None:
+        # Some pages are in development
+        context.log.warning(f"Table not found for {name_raw}")
+        return
+
+    data = {}
+
+    for row in table.findall(".//tr"):
+        label_cell = row.find(".//td[@class='label']")
+        if label_cell is not None:
+            label = label_cell.text_content().strip().strip(":")
+            continue  # Move to the next row for the corresponding value
+
+        value_cell = row.find(".//td[@class='value']")
+        if value_cell is not None and label:
+            # Extract each div's text inside value_cell
+            values = []
+            for div in value_cell.findall(".//div"):
+                text = div.text_content().strip()
+                values.append(text)
+            data[label] = values
+            label = None  # Reset the label for the next row
+
+    crawl_person(context, data, href, name_raw)
+
+
+def crawl_person(context: Context, data, href, name_raw):
+    # name = data.pop("titul, meno, priezvisko") # name with title
+    # year = data.pop("oznámenie za rok")
+    position = data.pop("vykonávaná verejná funkcia")
+    person = context.make("Person")
+    person.id = context.make_id(name_raw)
```

Comment:
I think we need more disambiguating information in the ID than just name, for such a broad pool of people.

Perhaps Internal number is consistent for a given person each year? e.g. ZP235369
Just confirm.

The alternative is urlparse-ing the URL to get userid e.g. Peter.Demcak, but I think using internal ID is fine since they're publishing it front and centre.

Reply:
yes, internal number is consistent for a given person each year, just added it to the id